core/client/fs/src/test/java/alluxio/client/block/policy/options/GetWorkerOptionsTest.java

```diff
@@ -25,6 +25,8 @@
 import java.util.List;
 import java.util.Random;
 
+import static org.junit.Assert.assertEquals;
```

Comment:
can you put `import static org.junit.Assert.assertEquals;` in front of the other imports?